xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -538,14 +539,18 @@ INFO_RET CMusicInfoScanner::ScanTags(const CFileItemList& items, CFileItemList&
     }
 
     if (m_handle && m_itemCount>0)
-      m_handle->SetPercentage(m_currentItem/(float)m_itemCount*100);
+      m_handle->SetPercentage(m_currentItem / (float)m_itemCount * 100);
 
     if (!tag.Loaded())
     {
       CLog::Log(LOGDEBUG, "%s - No tag found for: %s", __FUNCTION__, pItem->GetPath().c_str());
       continue;
     }
-    scannedItems.Add(pItem);
+    pItem->LoadEmbeddedCue();
```

Comment:
I asked you to check the tag first. the code is hard to read this way. why jumping into a method called "LoadEmbeddedCue" if there is no embedded cue?